src/com/epam/javacourse/homework4/Main.java

```diff
@@ -1,11 +1,14 @@
 package com.epam.javacourse.homework4;
 
+import java.nio.file.Path;
 import java.util.List;
 import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) {
-        GreenhouseManagement greenhouse = new GreenhouseManagement();
+
+    public static void main(String[] args) throws InvalidChoiceException {
+        Path path = Path.of("C:\\Users\\Sviatlana_Shedava\\IdeaProjects\\SviatlanaShedava5\\src\\com\\epam\\javacourse\\homework4\\plants.txt");
```

Comment:
This path will never work for me, cause I don't have `C:\\Users\\Sviatlana_Shedava\\` directory.
To make it generic for all users it's better to take a look at 'relative paths' and use one here.